react/src/components/UserDropdownMenu.tsx

```diff
@@ -0,0 +1,156 @@
+import { useSuspendedBackendaiClient } from '../hooks';
+import { useCurrentUserInfo } from '../hooks/backendai';
+import { useTanQuery } from '../hooks/reactQueryAlias';
+import { useWebComponentInfo } from './DefaultProviders';
+import Flex from './Flex';
+import {
+  UserOutlined,
+  MailOutlined,
+  SecurityScanOutlined,
+  ExclamationCircleOutlined,
+  LockOutlined,
+  HolderOutlined,
+  FileTextOutlined,
+  LogoutOutlined,
+} from '@ant-design/icons';
+import { useDebounce, useToggle } from 'ahooks';
```

Comment:
```suggestion
import { useDebounce } from 'ahooks';
```

please remove unused import :)